src/test/java/ru/training/at/hw7/tests/ex1/ExerciseOneOfHw7Test.java

```diff
@@ -0,0 +1,31 @@
+package ru.training.at.hw7.tests.ex1;
+
+import static ru.training.at.hw7.SiteJdi.homePage;
+import static ru.training.at.hw7.SiteJdi.metalAndColorsPage;
+
+import org.testng.annotations.Test;
+import ru.training.at.hw7.dataProvider.ParametersForCheckingMetalAndColorsForm;
+import ru.training.at.hw7.dataProvider.MetalsColorsDataSet;
+import ru.training.at.hw7.tests.BaseTest;
+
+public class ExerciseOneOfHw7Test extends BaseTest {
+
+
+    @Test (dataProviderClass = MetalsColorsDataSet.class, dataProvider = "metalsColorsData")
+    public void doExerciseOneInHw7(ParametersForCheckingMetalAndColorsForm parameters) {
+        //User should be logged in
+        softAssert.assertEquals(homePage.getUsername(), properties.getUsername());
```

Comment:
Soft assert can be used only if we can check some more conditions and fail after.
In this case we cann't open another page without this condition, so we cann't use soft assert.

Reply:
Replaced for hard Assert.